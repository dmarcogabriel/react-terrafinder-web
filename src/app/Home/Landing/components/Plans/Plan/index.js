import React from 'react';
import cn from 'classnames';
import { MdClose, MdCheck } from 'react-icons/md';
import {
  container,
  header,
  title,
  plan as planStyle,
  content,
  buttonStyle,
  planKind,
  featureStyle,
  featureName,
  iconStyle,
} from './Plan.module.scss';

export default function Plan({ plan, i }) {
  return (
    <div className={container} style={{ zIndex: i === 1 ? 500 : 0 }}>
      <div className={header}>
        <p className={cn(`text-${plan.color}`, title)}>{plan.amount}</p>

        <p className={planStyle}>{plan.plan}</p>
      </div>

      <div className={cn(`bg-${plan.bgColor}`, content)}>
        <p className={planKind}>{plan.kind}</p>

        <ul>
          {plan.features.map((feature) => (
            <li key={feature.id} className={featureStyle}>
              {feature.included ? (
                <MdCheck className={iconStyle} />
              ) : (
                <MdClose className={iconStyle} />
              )}
              <span className={featureName}>{feature.name}</span>
            </li>
          ))}
        </ul>

        <button className={cn(`text-${plan.color}`, buttonStyle)} type="button">
          Quero esse
        </button>
      </div>
    </div>
  );
}
