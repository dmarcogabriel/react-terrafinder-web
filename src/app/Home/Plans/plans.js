export default [
  {
    id: 'p1',
    amount: 'Grátis',
    kind: 'Básico',
    color: 'lime-green',
    bgColor: 'lime-green',
    features: [
      {
        id: 'f1',
        name: 'Duração de 7 dias',
        included: true,
      },
      {
        id: 'f2',
        name: 'Até 15 fotos',
        included: true,
      },
      {
        id: 'f3',
        name: 'Sem demarcação do imóvel no mapa',
        included: false,
      },
      {
        id: 'f4',
        name: 'Sem suporte',
        included: false,
      },
    ],
  },
  {
    id: 'p2',
    amount: 'R$ 50',
    plan: 'por anúncio',
    kind: 'PREMIUM',
    color: 'yellow',
    bgColor: 'yellow',
    features: [
      {
        id: 'f1',
        name: 'Duração do anúncio de 90 dias',
        included: true,
      },
      {
        id: 'f2',
        name: 'Visibilidade aumentada na plataforma',
        included: true,
      },
      {
        id: 'f3',
        name: 'Até 30 fotos e 1 vídeo',
        included: true,
      },
      {
        id: 'f4',
        name: 'Sem filmagens por drone nem tratamento de fotos',
        included: false,
      },
    ],
  },
  {
    id: 'p3',
    amount: 'A definir',
    plan: 'Faça uma cotação com nossos especialistas',
    kind: 'Profissional',
    color: 'pink',
    bgColor: 'blue',
    features: [
      {
        id: 'f1',
        name: 'Duração de até 365 dias',
        included: true,
      },
      {
        id: 'f2',
        name: 'Maior visibilidade da plataforma',
        included: true,
      },
      {
        id: 'f3',
        name: 'Gravação da propriedade com drones e tratamento de imagens',
        included: false,
      },
      {
        id: 'f4',
        name: 'Suporte exclusivo',
        included: false,
      },
    ],
  },
];
