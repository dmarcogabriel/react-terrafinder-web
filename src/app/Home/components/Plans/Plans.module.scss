@import 'styles';

.container {
  .title {
    font-weight: normal;
    font-size: 2rem;
    margin: 2rem 0;
    text-align: center;
  }

  .subtitle {
    font-weight: normal;
    font-size: 1.25rem;
    margin: 2rem 0;
    text-align: center;
  }

  .plans {
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;

    @media ($desktop) {
      flex-direction: row;
    }
  }

  .plansDescription {
    padding: 0 2rem;
    margin: 1rem 0 2rem;

    @media ($desktop) {
      padding: 0 10rem;
    }

    > p {
      margin: 1rem 0;
      color: $gray-dark;
      line-height: 2rem;
    }
  }
}
