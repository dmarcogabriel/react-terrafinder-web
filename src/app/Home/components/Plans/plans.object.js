import { purple } from '@mui/material/colors';

export const PLANS = [
  {
    id: 'p1',
    amount: 'Grátis',
    kind: 'Básico',
    color: '#32cd8b',
    bgColor: '#32cd8b',
    link: '/',
    features: [
      {
        id: 'f1',
        name: 'Duração de 7 dias',
        included: true,
      },
      {
        id: 'f2',
        name: 'Até 15 fotos',
        included: true,
      },
      {
        id: 'f3',
        name: 'Sem demarcação do imóvel no mapa',
        included: false,
      },
      {
        id: 'f4',
        name: 'Sem suporte',
        included: false,
      },
    ],
  },
  {
    id: 'p2',
    amount: 'R$ 129,00',
    plan: 'por anúncio',
    kind: 'PREMIUM',
    color: '#cd8632',
    bgColor: '#cd8632',
    link: '/',
    features: [
      {
        id: 'f1',
        name: 'Duração do anúncio de 90 dias',
        included: true,
      },
      {
        id: 'f2',
        name: 'Visibilidade aumentada na plataforma',
        included: true,
      },
      {
        id: 'f3',
        name: 'Até 30 fotos e 1 vídeo',
        included: true,
      },
      {
        id: 'f4',
        name: 'Sem filmagens por drone nem tratamento de fotos',
        included: false,
      },
    ],
  },
  {
    id: 'p3',
    amount: 'A definir',
    plan: 'Faça uma cotação com nossos especialistas',
    kind: 'Profissional',
    color: purple[300],
    bgColor: ' #3279cd',
    link: '/contact-us',
    features: [
      {
        id: 'f1',
        name: 'Duração de até 365 dias',
        included: true,
      },
      {
        id: 'f2',
        name: 'Maior visibilidade da plataforma',
        included: true,
      },
      {
        id: 'f3',
        name: 'Gravação da propriedade com drones e tratamento de imagens',
        included: true,
      },
      {
        id: 'f4',
        name: 'Suporte exclusivo',
        included: true,
      },
    ],
  },
];
