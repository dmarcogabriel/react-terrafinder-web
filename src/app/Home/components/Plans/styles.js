import React from 'react';
import { styled } from '@mui/material/styles';
import { Typography, Box } from '@mui/material';

export const PlansTitle = styled((props) => (
  <Typography variant="h4" {...props} />
))({
  textAlign: 'center',
  fontWeight: 'bolder',
});

export const PlansSubtitle = styled((props) => (
  <Typography variant="h5" {...props} />
))({
  textAlign: 'center',
});

export const PlansContent = styled(Box)(({ theme }) => ({
  display: 'flex',
  flexDirection: 'column',
  justifyContent: 'center',
  alignItems: 'center',
  [theme.breakpoints.up('md')]: {
    flexDirection: 'row',
  },
}));

export const PlansDescriptionText = styled(Typography)({
  color: '#4b4b4b',
  lineHeight: '2rem',
});
