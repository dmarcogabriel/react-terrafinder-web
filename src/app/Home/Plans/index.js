import React from 'react';
import classes from './Plans.module.scss';
import Plan from './Plan';
import PLANS from './plans';

export default function Plans() {
  return (
    <div className={classes.container}>
      <h1 className={classes.title}>Conheça nosso planos de anúncio</h1>
      <h2 className={classes.subtitle}>
        Escolha o que melhor se encaixa para você
      </h2>

      <div className={classes.plans}>
        {PLANS.map((plan, i) => (
          <Plan key={plan.id} plan={plan} i={i} />
        ))}
      </div>
    </div>
  );
}
