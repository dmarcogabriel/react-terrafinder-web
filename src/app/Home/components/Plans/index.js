import React from 'react';
import classes from './Plans.module.scss';
import Plan from './Plan';
import PLANS from './plans';

export default function Plans() {
  return (
    <div className={classes.container}>
      <h1 className={classes.title}>Conheça nosso planos de anúncio</h1>
      <h2 className={classes.subtitle}>
        Escolha o que melhor se encaixa para você
      </h2>

      <div className={classes.plans}>
        {PLANS.map((plan, i) => (
          <Plan key={plan.id} plan={plan} i={i} />
        ))}
      </div>

      <div className={classes.plansDescription}>
        <p>
          <b>Gratuito:</b> Recomendado para iniciantes, é uma ótima forma de
          testar a plataforma. Você pode testar a plataforma. Você pode criar um
          anúncio gratuíto da sua propriedade, com as informações gerais do
          imóvel, e dura 7 dias.
        </p>

        <p>
          <b>Premium:</b> Recomendado para quem já tem conteúdos da propriedade
          como fotos, detalhes do imóvel como tipo de solo, atividades,
          topografia. Ele custa R$ 50,00 e seu anúncio ganha mais visibilidade
          dentro da plataforma.
        </p>

        <p>
          <b>Proffissional:</b> Entre em contato conosco para conversarmos e
          conhecermos melhor o seu imóvel rural. Iremos até a propriedade,
          colheremos as informações mais importanjtes dela e faremos excelentes
          gravações explorando o imóvel através de dones de ponta. Essa é a
          maneira mais fácil de vender.
        </p>
      </div>
    </div>
  );
}
