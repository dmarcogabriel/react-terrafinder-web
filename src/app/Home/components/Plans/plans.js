import React from 'react';
import { Box } from '@mui/material';
import Plan from './Plan';
import { PLANS } from './plans.object';
import {
  PlansTitle,
  PlansSubtitle,
  PlansContent,
  PlansDescriptionText,
} from './styles';

export const Plans = () => (
  <Box sx={{ py: 2, px: 2 }}>
    <PlansTitle sx={{ my: 2 }}>Conheça nossos planos de anúncio</PlansTitle>
    <PlansSubtitle sx={{ my: 2 }}>
      Escolha o que melhor se encaixa para você
    </PlansSubtitle>

    <PlansContent>
      {PLANS.map((plan, i) => (
        <Plan key={plan.id} plan={plan} i={i} />
      ))}
    </PlansContent>

    <Box
      sx={{
        px: { xs: 2, md: 10 },
        mt: 1,
        mb: 2,
      }}
    >
      <PlansDescriptionText sx={{ my: 2 }}>
        <b>Gratuito:</b> Recomendado para iniciantes, é uma ótima forma de
        testar a plataforma. Você pode testar a plataforma. Você pode criar um
        anúncio gratuíto da sua propriedade, com as informações gerais do
        imóvel, e dura 7 dias.
      </PlansDescriptionText>

      <PlansDescriptionText sx={{ my: 2 }}>
        <b>Premium:</b> Recomendado para quem já tem conteúdos da propriedade
        como fotos, detalhes do imóvel como tipo de solo, atividades,
        topografia. Ele custa R$ 50,00 e seu anúncio ganha mais visibilidade
        dentro da plataforma.
      </PlansDescriptionText>

      <PlansDescriptionText sx={{ my: 2 }}>
        <b>Profissional:</b> Entre em contato conosco para conversarmos e
        conhecermos melhor o seu imóvel rural. Iremos até a propriedade,
        colheremos as informações mais importanjtes dela e faremos excelentes
        gravações explorando o imóvel através de dones de ponta. Essa é a
        maneira mais fácil de vender.
      </PlansDescriptionText>
    </Box>
  </Box>
);
