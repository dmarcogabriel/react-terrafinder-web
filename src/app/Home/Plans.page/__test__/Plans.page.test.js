import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import { PlansPage } from '../Plans.page';

const Comp = (props) => (
  <BrowserRouter>
    <PlansPage {...props} />
  </BrowserRouter>
);

let mockUser;
const mockLogout = jest.fn();

jest.mock('hooks/useUser', () => ({
  useUser: () => ({
    currentUser: mockUser,
    userIsLogged: jest.fn(() => !!mockUser),
    logout: mockLogout,
  }),
}));

jest.mock('hooks/useNotification', () => ({
  useNotification: () => ({
    showNotification() {},
  }),
  NOTIFICATION_TYPES: { SUCCESS: 'success', ERROR: 'error' },
}));

describe('app/Home/Plans.page', () => {
  it('should render correctly', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Comp />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('should match snapshot', () => {
    const tree = renderer.create(<Comp />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
