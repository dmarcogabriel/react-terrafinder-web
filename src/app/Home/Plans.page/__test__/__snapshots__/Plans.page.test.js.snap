// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app/Home/Plans.page should match snapshot 1`] = `
Array [
  .c0 {
  height: 4rem;
  cursor: pointer;
}

<header
    className="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionFixed mui-fixed css-zhueze-MuiPaper-root-MuiAppBar-root"
    data-testid="app-bar"
  >
    <div
      className="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-nb5wfo-MuiToolbar-root"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        <img
          alt="logo"
          className="c0"
          src="\\"Logo.png\\""
        />
      </a>
      <button
        className="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeLarge css-mf1cb5-MuiButtonBase-root-MuiIconButton-root"
        data-testid="menuButton"
        disabled={false}
        onBlur={[Function]}
        onClick={[Function]}
        onContextMenu={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
          data-testid="MenuIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
          />
        </svg>
      </button>
    </div>
    <div
      className="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-mzps2i-MuiToolbar-root"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        <img
          alt="logo desktop"
          className="c0"
          src="\\"Logo.svg\\""
        />
      </a>
      <button
        className="MuiButton-root MuiButton-contained MuiButton-containedWarning MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root css-4umy8a-MuiButtonBase-root-MuiButton-root"
        disabled={false}
        onBlur={[Function]}
        onContextMenu={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <p
          className="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
        >
          Fale conosco!!
        </p>
      </button>
    </div>
  </header>,
  <div
    className="MuiBox-root css-1c9l5hu"
  />,
  <div>
    <h1>
      Conheça nosso planos de anúncio
    </h1>
    <h2>
      Escolha o que melhor se encaixa para você
    </h2>
    <div>
      <div
        style={
          Object {
            "zIndex": 0,
          }
        }
      >
        <div>
          <p
            className="text-lime-green"
          >
            Grátis
          </p>
          <p />
        </div>
        <div
          className="bg-lime-green"
        >
          <p>
            Básico
          </p>
          <ul>
            <li>
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M9 16.17L4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z"
                />
              </svg>
              <span>
                Duração de 7 dias
              </span>
            </li>
            <li>
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M9 16.17L4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z"
                />
              </svg>
              <span>
                Até 15 fotos
              </span>
            </li>
            <li>
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                />
              </svg>
              <span>
                Sem demarcação do imóvel no mapa
              </span>
            </li>
            <li>
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                />
              </svg>
              <span>
                Sem suporte
              </span>
            </li>
          </ul>
          <button
            className="text-lime-green"
            type="button"
          >
            Quero esse
          </button>
        </div>
      </div>
      <div
        style={
          Object {
            "zIndex": 500,
          }
        }
      >
        <div>
          <p
            className="text-yellow"
          >
            R$ 50
          </p>
          <p>
            por anúncio
          </p>
        </div>
        <div
          className="bg-yellow"
        >
          <p>
            PREMIUM
          </p>
          <ul>
            <li>
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M9 16.17L4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z"
                />
              </svg>
              <span>
                Duração do anúncio de 90 dias
              </span>
            </li>
            <li>
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M9 16.17L4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z"
                />
              </svg>
              <span>
                Visibilidade aumentada na plataforma
              </span>
            </li>
            <li>
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M9 16.17L4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z"
                />
              </svg>
              <span>
                Até 30 fotos e 1 vídeo
              </span>
            </li>
            <li>
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                />
              </svg>
              <span>
                Sem filmagens por drone nem tratamento de fotos
              </span>
            </li>
          </ul>
          <button
            className="text-yellow"
            type="button"
          >
            Quero esse
          </button>
        </div>
      </div>
      <div
        style={
          Object {
            "zIndex": 0,
          }
        }
      >
        <div>
          <p
            className="text-pink"
          >
            A definir
          </p>
          <p>
            Faça uma cotação com nossos especialistas
          </p>
        </div>
        <div
          className="bg-blue"
        >
          <p>
            Profissional
          </p>
          <ul>
            <li>
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M9 16.17L4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z"
                />
              </svg>
              <span>
                Duração de até 365 dias
              </span>
            </li>
            <li>
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M9 16.17L4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z"
                />
              </svg>
              <span>
                Maior visibilidade da plataforma
              </span>
            </li>
            <li>
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                />
              </svg>
              <span>
                Gravação da propriedade com drones e tratamento de imagens
              </span>
            </li>
            <li>
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                />
              </svg>
              <span>
                Suporte exclusivo
              </span>
            </li>
          </ul>
          <button
            className="text-pink"
            type="button"
          >
            Quero esse
          </button>
        </div>
      </div>
    </div>
    <div>
      <p>
        <b>
          Gratuito:
        </b>
         Recomendado para iniciantes, é uma ótima forma de testar a plataforma. Você pode testar a plataforma. Você pode criar um anúncio gratuíto da sua propriedade, com as informações gerais do imóvel, e dura 7 dias.
      </p>
      <p>
        <b>
          Premium:
        </b>
         Recomendado para quem já tem conteúdos da propriedade como fotos, detalhes do imóvel como tipo de solo, atividades, topografia. Ele custa R$ 50,00 e seu anúncio ganha mais visibilidade dentro da plataforma.
      </p>
      <p>
        <b>
          Proffissional:
        </b>
         Entre em contato conosco para conversarmos e conhecermos melhor o seu imóvel rural. Iremos até a propriedade, colheremos as informações mais importanjtes dela e faremos excelentes gravações explorando o imóvel através de dones de ponta. Essa é a maneira mais fácil de vender.
      </p>
    </div>
  </div>,
  <footer
    className="MuiBox-root css-108tvhz"
  >
    <hr
      className="MuiDivider-root MuiDivider-fullWidth css-tz2ti8-MuiDivider-root"
    />
    <div
      className="MuiBox-root css-zitybv"
    >
      <h3
        className="MuiTypography-root MuiTypography-h3 css-15anx50-MuiTypography-root"
      >
        Terrafinder
      </h3>
      <ul
        className="MuiList-root MuiList-padding css-12n6fjr-MuiList-root"
      >
        <h6
          className="MuiTypography-root MuiTypography-h6 css-xqtpgv-MuiTypography-root"
        >
          Institucional
        </h6>
        <li
          className="MuiListItem-root MuiListItem-gutters css-1nxmd3h-MuiListItem-root"
          disabled={false}
        >
          <div
            className="MuiListItemButton-root MuiListItemButton-gutters MuiButtonBase-root css-cvhtoe-MuiButtonBase-root-MuiListItemButton-root"
            data-testid="link-t1-l1"
            onBlur={[Function]}
            onClick={[Function]}
            onContextMenu={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            Página Inicial
          </div>
        </li>
        <li
          className="MuiListItem-root MuiListItem-gutters css-1nxmd3h-MuiListItem-root"
          disabled={false}
        >
          <div
            className="MuiListItemButton-root MuiListItemButton-gutters MuiButtonBase-root css-cvhtoe-MuiButtonBase-root-MuiListItemButton-root"
            data-testid="link-t1-l2"
            onBlur={[Function]}
            onClick={[Function]}
            onContextMenu={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            Quem somos
          </div>
        </li>
        <li
          className="MuiListItem-root MuiListItem-gutters css-1nxmd3h-MuiListItem-root"
          disabled={false}
        >
          <div
            className="MuiListItemButton-root MuiListItemButton-gutters MuiButtonBase-root css-cvhtoe-MuiButtonBase-root-MuiListItemButton-root"
            data-testid="link-t1-l3"
            onBlur={[Function]}
            onClick={[Function]}
            onContextMenu={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            Política de privacidade
          </div>
        </li>
        <li
          className="MuiListItem-root MuiListItem-gutters css-1nxmd3h-MuiListItem-root"
          disabled={false}
        >
          <div
            className="MuiListItemButton-root MuiListItemButton-gutters MuiButtonBase-root css-cvhtoe-MuiButtonBase-root-MuiListItemButton-root"
            data-testid="link-t1-l4"
            onBlur={[Function]}
            onClick={[Function]}
            onContextMenu={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            Fale Conosco
          </div>
        </li>
      </ul>
      <ul
        className="MuiList-root MuiList-padding css-12n6fjr-MuiList-root"
      >
        <h6
          className="MuiTypography-root MuiTypography-h6 css-xqtpgv-MuiTypography-root"
        >
          Funcionalidades
        </h6>
        <li
          className="MuiListItem-root MuiListItem-gutters css-1nxmd3h-MuiListItem-root"
          disabled={false}
        >
          <div
            className="MuiListItemButton-root MuiListItemButton-gutters MuiButtonBase-root css-cvhtoe-MuiButtonBase-root-MuiListItemButton-root"
            data-testid="link-t2-l1"
            onBlur={[Function]}
            onClick={[Function]}
            onContextMenu={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            Quero anunciar meu imóvel
          </div>
        </li>
        <li
          className="MuiListItem-root MuiListItem-gutters css-1nxmd3h-MuiListItem-root"
          disabled={false}
        >
          <div
            className="MuiListItemButton-root MuiListItemButton-gutters MuiButtonBase-root css-cvhtoe-MuiButtonBase-root-MuiListItemButton-root"
            data-testid="link-t2-l2"
            onBlur={[Function]}
            onClick={[Function]}
            onContextMenu={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            Quero comprar um imóvel
          </div>
        </li>
        <li
          className="MuiListItem-root MuiListItem-gutters css-1nxmd3h-MuiListItem-root"
          disabled={false}
        >
          <div
            className="MuiListItemButton-root MuiListItemButton-gutters MuiButtonBase-root css-cvhtoe-MuiButtonBase-root-MuiListItemButton-root"
            data-testid="link-t2-l3"
            onBlur={[Function]}
            onClick={[Function]}
            onContextMenu={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            Preços e planos
          </div>
        </li>
      </ul>
      <h6
        className="MuiTypography-root MuiTypography-h6 css-xqtpgv-MuiTypography-root"
      >
        Fique por dentro das novidades!
      </h6>
      <p
        className="MuiTypography-root MuiTypography-body1 css-15wu8wo-MuiTypography-root"
      >
        Se inscreva em nossa newsletter para receber os anúncios com os melhores preços!
      </p>
      <form
        className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-np4jd-MuiPaper-root"
        noValidate={true}
        onSubmit={[Function]}
      >
        <div
          className="MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-fullWidth css-13bnupw-MuiInputBase-root"
          onClick={[Function]}
        >
          <input
            aria-invalid={false}
            className="MuiInputBase-input css-yz9k0d-MuiInputBase-input"
            data-testid="emailInput"
            inputMode="email"
            onAnimationStart={[Function]}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="Informe seu e-mail aqui"
            type="text"
            value=""
          />
        </div>
        <button
          className="MuiButton-root MuiButton-contained MuiButton-containedSuccess MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root css-1h0hyhf-MuiButtonBase-root-MuiButton-root"
          data-testid="registerEmail"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onContextMenu={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          Cadastrar
        </button>
      </form>
    </div>
    <hr
      className="MuiDivider-root MuiDivider-fullWidth css-tz2ti8-MuiDivider-root"
    />
    <h6
      className="MuiTypography-root MuiTypography-subtitle2 css-qwq8wp-MuiTypography-root"
    >
      Copyright C 2020 Terrafinder. Todos os direitos reservados.
    </h6>
  </footer>,
]
`;
